docs/rollout/appdev-devex.md

```diff
@@ -0,0 +1,16 @@
+---
+title: AppDev and DevEx
+layout: rollout
+---
+
+## Emerging need for DevEx
+
+In recent decades, business trends have quickly elevated the demand for effective AppDev (application development). And technology leaders know that empowering AppDevs (application developers) is thus critical for business success. In the past AppDev teams would support themselves. A team lead or an engineering manager would procure dev tools, iterate on process, and otherwise manage the support structure of the team so that they are motivated to do good work, and ideally continuously improving. In other cases, an IT admin would deploy standardized dev tools to teams upon request. But more recently, companies have realized this is still insufficient:
```

Comment:
A lot of this is about "DevEx is a new role/you have to invest more".

I think it's more important to emphasize "DevEx has always been something teams do, though they may not have staffed it or acknowledged it".